CMakeLists.txt

```diff
@@ -181,14 +190,6 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
 if(USE_CUDA)
   find_package(CUDA REQUIRED)
```

Comment:
one problem was without find_package(CUDA), headers weren’t being found for compile.